### ISSUE TITLE
AVT 3 - Tile Selectable: VoiceOver does not announce when a selectable tile is checked or unchecked

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2484]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
With VoiceOver enabled tab to the selectable tile that is checked (selectable tile 2)

## Expected Result
VoiceOver announces when the selectable tile is checked and unchecked. 

Note: Based on the selectable tiles type="radio" the selectable component should follow the [ARIA Authoring Practices 1.1 design pattern for radio group](https://www.w3.org/TR/wai-aria-practices-1.1/examples/radio/radio-1/radio-1.html) which requires  aria-checked attribute set on the selectable tiles.

**Comment by snidersd** [comment]
@dakahn This issue needs to be labeled A11y and React.

**Comment by snidersd** [comment]
@dakahn Retested and this is still an issue.

**Comment by abbeyhrt** [comment]
Hi @snidersd! I am not able to reproduce this problem with VO on Safari, would you be able to check for me to see if this is still a problem you're encountering?

**Comment by snidersd** [comment]
@abbeyhrt Retested and I no longer see the issue.

**Comment by abbeyhrt** [comment]
Okay great, thanks for looking into it! I'm going to close this issue since it has been addressed 👍
